scriptworker/task.py

```diff
@@ -66,6 +67,36 @@ def get_run_id(claim_task):
     return claim_task['runId']
 
 
+# get_action_name {{{1
+def get_action_name(task):
+    """Get the name of an action task.
+
+    Args:
+        obj (ChainOfTrust or LinkOfTrust): the trust object to inspect
+
+    Returns:
+        str: the name.
+
+    """
+    name = task['extra'].get('action', {}).get('name')
+    return name
+
+
+# get_commit_message {{{1
+def get_commit_message(task):
+    """Get the commit message for a task.
+
+    Args:
+        obj (ChainOfTrust or LinkOfTrust): the trust object to inspect
+
+    Returns:
+        str: the commit message.
+
+    """
+    msg = task['payload'].get('env', {}).get('GECKO_COMMIT_MSG', ' ')
```

Comment:
No, we default to a blank space because non-try decision tasks default to a blank space. Then I learned that cron tasks use '' as the comment, and hardcoded that elsewhere.